packages/components/select-v2/src/useSelect.ts

```diff
@@ -725,20 +725,31 @@ const useSelect = (props: ISelectV2Props, emit) => {
 
       return option
     }
+    let label = value
+    if (cachedOptions && cachedOptions.length) {
+      const option = cachedOptions.find(
+        (option) => getValueKey(getValue(option)) === selectValue
+      )
+      if (option) {
+        label = getLabel(option)
```

Comment:
```suggestion
        return option
```

Why not just return the `option` directly

Reply:
It seems to be OK, I fell into the misunderstanding of the previous code.